conans/client/printer.py

```diff
@@ -25,10 +26,16 @@ def __init__(self, out):
 
     def print_inspect(self, inspect):
         for k, v in inspect.items():
-            if isinstance(v, dict):
-                self._out.writeln("%s" % k)
-                for sk, sv in sorted(v.items()):
-                    self._out.writeln("    %s: %s" % (sk, str(sv)))
+            if (k == "default_options" and v is not None) or isinstance(v, dict):
```

Comment:
I hadn't this in mind, but to decouple model-specific from generic. Something like:

```python
for k, v in inspect.items():
    # corner cases, model specific
    if k == "default_options":
          if isinstance(v, str):
              v = OptionsValues.loads(v).items()
    # start generic code
    if isinstance(v, dict):
       ...
```

The reason why I find the current proposal a bit weird, is that you are calling and printing "options_values", to anything that is a dict, not only "options_values", e.g. like "options" or maybe "settings" if they are specified as dict.

Reply:
`options_values` is different because of values assigned to the option. Also, this crazy statement was created to format any value as dict only when have multiple lines

Reply:
done, please review again.